casepropods/family_connect_registration/tests.py

```diff
@@ -0,0 +1,136 @@
+                "id": "b03d0ba0-3baf-4acb-9943-f9ff89ef2412",
+                "stage": "postbirth",
+                "mother_id": "test_id",
+                "validated": True,
+                "data": {
+                    "hoh_surname": "the builder",
+                    "hoh_id": "hoh00001-63e2-4acc-9b94-26663b9bc267",
+                    "receiver_id": "hoh00001-63e2-4acc-9b94-26663b9bc267",
+                    "operator_id": "hcw00001-63e2-4acc-9b94-26663b9bc267",
+                    "language": "eng_UG",
+                    "msg_type": "text",
+                    "msg_receiver": "head_of_household",
+                    "mama_id_no": "12345",
+                    "last_period_date": "20150202",
+                    "mama_surname": "zin",
+                    "mama_id_type": "ugandan_id",
+                    "hoh_name": "bob",
+                    "mama_name": "sue"},
+                "source": 1,
+                "created_at": "2016-07-27T15:41:55.102172Z",
+                "updated_at": "2016-07-27T15:41:55.102200Z",
+                "created_by": 1,
+                "updated_by": 1
+            }]}
+        return (200, headers, json.dumps(resp))
+
+    @responses.activate
+    def test_read_data_no_registrations(self):
+        # Add callback
+        responses.add_callback(
+            responses.GET, self.url,
+            callback=self.registration_callback_no_matches,
+            match_querystring=True, content_type="application/json")
+
+        result = self.pod.read_data({'case_id': self.case.id})
+
+        self.assertEqual(result, {"items": []})
+
+    @responses.activate
+    def test_read_data_one_registration(self):
+        # Add callback
+        responses.add_callback(
+            responses.GET, self.url,
+            callback=self.registration_callback_one_match,
+            match_querystring=True, content_type="application/json")
+
+        result = self.pod.read_data({'case_id': self.case.id})
+
+        self.assertEqual(result, {"items": [
+            {"name": "Mother Name", "value": "sue"},
+            {"name": "Mother Surname", "value": "zin"},
+            {"name": "Date of last period", "value": "20150202"},
+            {"name": "Language Preference", "value": "eng_UG"},
+            {"name": "ID Type", "value": "ugandan_id"},
+            {"name": "ID Number", "value": "12345"},
+            {"name": "Message Receiver", "value": "head_of_household"},
+            {"name": "Receiver ID", "value":
+                "hoh00001-63e2-4acc-9b94-26663b9bc267"},
+            {"name": "Head of Household Name", "value": "bob"},
+            {"name": "Head of Household Surname", "value": "the builder"},
+            {"name": "Head of Household ID",
+                "value": "hoh00001-63e2-4acc-9b94-26663b9bc267"},
+            {"name": "Operator ID", "value":
+                "hcw00001-63e2-4acc-9b94-26663b9bc267"},
+            {"name": "Receives Messages As", "value": "text"},
+            ]})
```

Comment:
We should probably also have one field that results in `Unknown`, so that we can test all 3 types (on in the root, one in `data`, and one that isn't in either). We probably also don't need this number of fields to test with, but making the tests look like the actual responses is probably a good idea.